### PR TITLE
Dmp 2591 remove audit search endpoint

### PR DESCRIPTION
**Pull request by mmmalik1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-2591

### Change description ###
Remove audit search GET endpoint

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerDownloadIntTest.java [hunk 1]
```diff
@@ -109,16 +104,6 @@ void audioRequestDownloadShouldDownloadFromOutboundStorageAndReturnSuccess() thr
...
-
-        AuditSearchQuery searchQuery = new AuditSearchQuery();
-        searchQuery.setCaseId(mediaRequestEntity.getHearing().getCourtCase().getId());
-        searchQuery.setFromDate(OffsetDateTime.now().minusDays(1));
-        searchQuery.setToDate(OffsetDateTime.now().plusDays(1));
-        searchQuery.setAuditActivityId(DOWNLOAD_AUDIT_ACTIVITY_ID);
-
-        List<AuditEntity> auditEntities = auditService.search(searchQuery);
-        assertEquals("2", auditEntities.get(0).getCourtCase().getCaseNumber());
-        assertEquals(1, auditEntities.size());
```
Comment (davet1985) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerDownloadIntTest.java:121]: This needs to be tested in a different way, not just removed.

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerPlaybackIntTest.java [hunk 2]
```diff
@@ -116,15 +111,6 @@ void audioRequestPlaybackShouldPlaybackFromOutboundStorageAndReturnSuccess() thr
...
 
-        AuditSearchQuery searchQuery = new AuditSearchQuery();
-        searchQuery.setCaseId(mediaRequestEntity.getHearing().getCourtCase().getId());
-        searchQuery.setFromDate(OffsetDateTime.now().minusDays(1));
-        searchQuery.setToDate(OffsetDateTime.now().plusDays(1));
-        searchQuery.setAuditActivityId(PLAYBACK_AUDIT_ACTIVITY_ID);
-
-        List<AuditEntity> auditEntities = auditService.search(searchQuery);
-        assertEquals("2", auditEntities.get(0).getCourtCase().getCaseNumber());
-        assertEquals(1, auditEntities.size());
```
Comment (davet1985) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerPlaybackIntTest.java:127]: Again, this should be tested in a different way.

File: src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java [hunk 3]
```diff
@@ -166,20 +161,6 @@ private void assertTranscriptionWorkflow(TranscriptionWorkflowEntity transcripti
         assertEquals(expectedWorkflowActor, transcriptionWorkflowToCheck.getWorkflowActor());
     }
 
-    private void assertAudit(int expected) {
```
Comment (davet1985) [inline src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java:169]: The deletions in here need to tested in a different way and not removed, as per the other comments.

**Review by davet1985 — CHANGES_REQUESTED** [review]
The assertions related to audits that have been deleted should be testing in a different way, not removed. We are still creating the audit entries in those scenarios.

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerDownloadIntTest.java (hunk 1 above)
Comment (mmmalik1) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerDownloadIntTest.java]: Code updated for tests

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerPlaybackIntTest.java (hunk 2 above)
Comment (mmmalik1) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerPlaybackIntTest.java]: Code updated for tests

File: src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java (hunk 3 above)
Comment (mmmalik1) [reply src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java:169]: Code updated for tests

**Review by davet1985 — APPROVED** [review]

**Pull request merged**